flink-autoscaler/pom.xml

```diff
@@ -57,6 +57,18 @@ under the License.
             <groupId>org.quartz-scheduler</groupId>
             <artifactId>quartz</artifactId>
             <version>${quartz.version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>HikariCP-java7</artifactId>
+                    <groupId>com.zaxxer</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
+            <groupId>com.zaxxer</groupId>
+            <artifactId>HikariCP</artifactId>
+            <version>${hikari.version}</version>
```

Comment:
Are you sure this dependency works with quartz? Did you test flink-kubernetes-operator with autoscaler?

Reply:
Thank you for your feedback. I'll be testing MySQL and PostgreSQL later, and I'm still tweaking it.

Reply:
>  I'll be testing MySQL and PostgreSQL later, and I'm still tweaking it.

So far, flink-kubernetes-operator doesn't support Jdbc state store and event handler. But autoscaler uses quartz, this comment isn't related to MySQL and PostgreSQL.

This comment means whether flink-kubernetes-operator with autoscaler works well when we introduce the other version for `HikariCP`.

Also, standalone-autoscaler will use the Jdbc state store and event handler, it's needed to test them.